files-csharp/FastestWayToReadATextFileInCsharp/FastestWayToReadATextFileInCsharp/WaysToReadATextFileInCsharp.cs

```diff
@@ -1,86 +1,128 @@
 ﻿using System.Text;
 
-namespace FastestWayToReadATextFileInCsharp
+namespace FastestWayToReadATextFileInCsharp;
+
+public class WaysToReadATextFileInCsharp
 {
-    public class WaysToReadATextFileInCsharp
+    private readonly string _sampleFilePath;
+
+    public WaysToReadATextFileInCsharp(string sampleFilePath)
+    {
+        _sampleFilePath = sampleFilePath;
+    }
+
+    public string UseFileReadAllLines()
     {
-        private readonly string _sampleFilePath;
+        var lines = File.ReadAllLines(_sampleFilePath);
+        var stringBuilder = new StringBuilder();
 
-        public WaysToReadATextFileInCsharp(string sampleFilePath)
+        foreach (var line in lines)
         {
-            _sampleFilePath = sampleFilePath;
+            stringBuilder.AppendLine(line);
         }
+        stringBuilder.Length -= Environment.NewLine.Length;
 
-        public string UseFileReadAllLines()
-        {
-            var stringArray = File.ReadAllLines(_sampleFilePath);
+        return stringBuilder.ToString();
+    }
 
-            return string.Join(Environment.NewLine, stringArray);
-        }
+    public string UseFileReadAllText() => File.ReadAllText(_sampleFilePath);
+    
+    public string UseFileReadLines()
+    {
+        var lines = File.ReadLines(_sampleFilePath);
```

Comment:
There is no need for this variable, the `File.ReadLines` call can be made in the `foreach`